engine/src/flutter/docs/contributing/Compiling-the-engine.md

```diff
@@ -92,15 +94,16 @@ host build available next to it: if you use `android_debug_unopt`, you should ha
 
 ### Compiling everything that matters on Linux
 
-The following script will update all the builds that matter if you're developing on Linux and testing on Android and created the `.gclient` file in `~/dev/engine`:
+The following script will update all the builds that matter if you're developing on Linux and testing on Android if your `.gclient` file is located at `~/dev/flutter/engine/.gclient`:
```

Comment:
I think the script _has_ to be in `~/dev/flutter/.gclient` actually.

Reply:
Good catch, fixed!